### DISCUSSION TITLE
BirdNet-Go: How do i save Clips on a Network Share?

### DISCUSSION THREAD

**Discussion by Slakish** [alexbelgium/hassio-addons#1602] [General]
Hello,

I have installed the BirdNet-Go addon. Unfortunately, all clips are gone after a restart. I just don't understand how to set the folders. When I mount an NFS share in HomeAssistant and enter it under “localdisk”, the clips are not saved. When I use the CIFS share, no connection to the NAS is established. So how can I save the clips on an NFS share that is mounted in Home Assistant or on a CIfs share? Unfortunately, this doesn't tell me anything:

The audio clips folder can be stored on an external or SMB drive by mounting it from the addon options, then specifying the path instead of "clips/". For example, "/mnt/NAS/Birdnet/" 

I enter my CIFS data and change the path “Clips” to /mnt/MyPath or just /MyPath?

**Comment by alexbelgium** [comment]
Hi, the log will tell you where the disk is mounted. In my example, I've mounted the local disk labeled "NAS" on which I've manually created a folder named "Birdnet" so my path is /mnt/NAS/Birdnet instead of clips/

**Reply by Slakish** [reply]
So if I want to use the share /share/Birdnet mounted in Home Assistant, this is right: [Screenshot_20241011_223347_Home Assistant](https://github.com/user-attachments/assets/040ef39a-b9f5-442e-8c55-0ec4c2584aad)

?

**Reply by alexbelgium** [reply]
Nearly : localdisk should contain the physical (ex /dev/sda1) or label of your usb drive (ex: Birdnet). Then it will be mounted to /mnt/sda1 or /mnt/Birdnet, and you still need to create a subfolder on your drive (ex:Subfolder). Your BirdSongs path you then be for example : /mnt/sda1/Subfolder. The log will show you the list of all available drives, including their physical names and labels

**Reply by Slakish** [reply]
I now have the subfolder "Clips" in share/BirdNet (share/Birdnet/Clips. So Local Disk would be "share/BirdNet" and the BirdSongs path "/mnt/Birdnet/Clips". However, the setting in the web interface remains at /Clips. correct ?

**Reply by alexbelgium** [reply]
No, localdisk must be the name of the drive. So if your hdd is labeled "Birdnet", you should add Birdnet in localdisk and /mnt/Birdnet/Clips in BirdSongs path. This would automatically replace "Clips/" in the interface. If it doesn't, just put /mnt/Birdnet/Clips in the place of Clips/. 

But if you have already mounted a drive to /share/Birdnet using HA's native mount system ; you should not use "localdisk". You should just mention "/share/Birdnet" in the BirdSongs option (which should overwrite the Clips/ path that is by default in the interface) or you manually replace the path in the interface

**Comment by Slakish** [comment]
Thank you very much. Now the clips are saved but you can no longer listen to them in the webui. But that must be a problem with BirdNet go.

**Reply by alexbelgium** [reply]
Half there ;-) Perhaps a permissions issues ?

**Reply by Slakish** [reply]
It can write, but not read. Maybe map the docker's power with that of the path?

**Reply by alexbelgium** [reply]
Mmh perhaps you could ask on the birdnet-go github ; but before is there nothing in logs ?

**Comment by Slakish** [comment]
Only this 

![Screenshot_20241012_135056_Home Assistant](https://github.com/user-attachments/assets/bc8610e4-3b65-4137-9ef0-9ea69882e81e)

**Reply by alexbelgium** [reply]
Looks like it used a double //! Probably a bug from birdnet-go then : it should accept absolute pathways (starting with /) and not only relative 

BTW thanks for the coffee!
